services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowValidationNegativeSpec.groovy

```diff
@@ -1,15 +1,20 @@
 package org.openkilda.functionaltests.spec.flows
 
 import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.messaging.error.MessageError
 import org.openkilda.messaging.info.event.IslInfoData
 import org.openkilda.messaging.model.FlowDto
 import org.openkilda.model.SwitchId
 import org.openkilda.northbound.dto.flows.FlowValidationDto
 import org.openkilda.testing.model.topology.TopologyDefinition.Switch
+import org.springframework.web.client.HttpClientErrorException
+
+import static org.openkilda.testing.Constants.NON_EXISTENT_FLOW_ID
```

Comment:
Import is in the wrong place. Please, install checkstyle stuff from here to fix it in the future.